packages/@o3r/pipeline/schematics/ng-add/templates/github/__dot__github/actions/setup/action.yml

```diff
@@ -6,16 +6,30 @@ runs:
   steps:
     - uses: actions/setup-node@0a44ba7841725637a19e28fa30b79a866c81b0a6 # v4.0.4
       with:
-        node-version: 20
-        cache: <%= packageManager %>
+        node-version: 20<% if (packageManager !== 'yarn') { %>
+        cache: <%= packageManager %><% } %>
     - name: Enable Corepack
       shell: bash
-      run: corepack enable
-    - name: Install
-<% if (npmRegistry) { %>
+      run: corepack enable<% if (packageManager === 'yarn') { %>
+    - name: Get yarn cache directory path
+      shell: bash
+      id: yarn-cache-dir-path
+      run: echo "dir=$(<% if (yarn2) { %>yarn config get cacheFolder<% } else { %>yarn cache dir<% } %>)" >> $GITHUB_OUTPUT
```

Comment:
Shouldn't we update our pipeline as well ?

Reply:
We could but with a feature request so that the setup action support npm, yarn 1, etc.
Definitely not in a fix on an RC branch.

Reply:
Can you open the request then please :)

Reply:
https://github.com/AmadeusITGroup/otter/issues/2425